### PR TITLE
Add TiliPool

### PR DESCRIPTION
**Pull request by wout**
This PR adds [TiliPool](https://tilipool.com/), the stake pool of [Tili.Gallery](https://tili.gallery/).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4305,5 +4305,19 @@
...
+    "ticker": "TILI",
+    "poolId": "4fbb9ac6b2c977ee76da892315fb1e26697a87bf8b54917efaa60053",
+    "social": {
+      "website": "https://tilipool.com/",
+      "twitter": "TiliPool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "TiliPool",
```
Comment (Bez625) [inline registry.json:4319]: Please can you change this to the TG handle of an individual user and not a group or channel? Thanks

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (wout) [reply registry.json:4319]: Is there a reason for not allowing a group or channel? Just curious. :)

Comment (Bez625) [reply registry.json:4319]: It's so we have an individual to reach out to in case we need to contact the pool

**Comment by wout** [comment]
I'm going to create a fresh PR.

**Pull request closed without merging**